app/graphql/mutations/create_order.rb

```diff
@@ -14,9 +19,6 @@ class CreateOrder < BaseMutation
         argument :input, Types::CreateOrderInputType, required: true
 
         field :order, Types::OrderType, null: false
-        field :order_quantities, [Types::OrderQuantityType], null: false
```

Comment:
I deleted this fields from the mutation response because I realized I could just specify them on the front-end within the `order` field